### ISSUE TITLE
Can we convert the tagname to camel case?

### ISSUE THREAD

**Issue by dotku** [NaturalIntelligence/fast-xml-parser#453]
* [x] Are you running the latest version?
* [ ] Have you included sample input, output, error, and expected output? 
* [x] Have you checked if you are using correct configuration?
* [ ] Did you try online [tool](https://naturalintelligence.github.io/fast-xml-parser/)? // can't visit

### Description
Want to display camelCase output

### Input

```xml
    <Equipment>
        <EquipmentHeader>
            <OemName>Hydrema</OemName>
            <Model>912F</Model>
        </EquipmentHeader>
    </Equipment>
```

### Output

Equipment.EquipmentHeader...

### Expected Output

equipment.equipmentHeader...

**Comment by amitguptagwl** [comment]
This feature is not implemented yet due the API designs for this requirement is finalize yet.

**Comment by erkie** [comment]
This can be closed now with the merged PR #469 😀
